app-emulation/euae/euae-0.8.29~wip4.recipe

```diff
@@ -71,11 +71,16 @@ BUILD_PREREQUIRES="
 	cmd:perl
 	cmd:pkg_config
 	"
+if [ "$effectiveTargetArchitecture" = x86_gcc2 ] ; then
```

Comment:
I think this, and the "--build=$EUAEBUILD" from the configure invocation not required at all.

Reply:
Req on x86_gcc2, for proper app build.

Reply:
Thats not true and not a proper solution. libtoolize should be used instead this before the configure invocation.

Reply:
libtoolize -fci
autoconf
automake --add-missing